### PR TITLE
Sfitz remove common checks

### PR DESCRIPTION
**Pull request by sorelfitzgibbon**
# Description
 - Removed `check_max` and `check_limits` in methods.config.  Changed calls to `check_max` to `check_limits` (external).
 - update submodules

## Testing Results
`nftest run a_mini-muse`
output: `/hot/software/pipeline/pipeline-call-sSNV/Nextflow/development/unreleased/sfitz-remove-common-checks/a_mini-muse`

NFTest asserts fail because `Expect` files have been updated based on the unmerged extract-sample-name branch

# Checklist

- [x] I have read the [code review guidelines](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3187646/Code+Review+Guidelines) and the [code review best practice on GitHub check-list](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3189956/Code+Review+Best+Practice+on+GitHub+-+Check+List).

- [x] I have reviewed the [Nextflow pipeline standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3193890/Nextflow+pipeline+standardization).

- [x] The name of the branch is meaningful and well formatted following the [standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3189956/Code+Review+Best+Practice+on+GitHub+-+Check+List), using \[AD_username (or 5 letters of AD if AD is too long)]-\[brief_description_of_branch].

- [x] I have set up or verified the branch protection rule following the [github standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3190380/GitHub+Standards#GitHubStandards-Branchprotectionrule) before opening this pull request.

- [x] I have added my name to the contributors listings in the ``manifest`` block in the `nextflow.config` as part of this pull request; I am listed already, or do not wish to be listed. (*This acknowledgement is optional.*)

- [x] I have added the changes included in this pull request to the `CHANGELOG.md` under the next release version or unreleased, and updated the date.

- [x] I have updated the version number in the `metadata.yaml` and `manifest` block of the `nextflow.config` file following [semver](https://semver.org/), or the version number has already been updated. (*Leave it unchecked if you are unsure about new version number and discuss it with the infrastructure team in this PR.*)

- [x] I have tested the pipeline on at least one A-mini sample.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Test CODEOWNERS
 # Default owner(s)
-*	@tyamaguchi-ucla @yashpatel6 @maotian06 @sorelfitzgibbon @uclahs-cds/nextflow-wg
+*	@tyamaguchi-ucla @yashpatel6 @maotian06 @sorelfitzgibbon @ucla-cds/nextflow-wg
```
Comment (sorelfitzgibbon) [inline .github/CODEOWNERS:3]: I haven't been able to get rid of this codeowners error.  It doesn't like either of the above.

Comment (yashpatel6) [reply .github/CODEOWNERS:3]: The issue was that particular team didn't have any access for the current repo; I've added `uclahs-cds/nextflow-wg` to the repo so the file should pass with that as an owner now

Comment (sorelfitzgibbon) [reply .github/CODEOWNERS]: Got it, reset to `@uclahs-cds/nextflow-wg`

**Review by yashpatel6 — APPROVED** [review]
Looks good!

**Pull request merged**